src/components/grid-aware/ImageHeader/ImageHeader.module.css

```diff
@@ -0,0 +1,121 @@
+/* Background bleed */
+
+.bleedWrapper {
+  display: grid;
+  grid-template-columns: 1fr minmax(var(--grid-content-min-width), var(--grid-content-max-width)) 1fr;
+  margin: auto;
+  max-width: var(--grid-bleed-max-width);
+}
+
+.bleedMainContent {
+  grid-column: 2;
+  grid-row: 1;
+  min-width: 0;
+}
+
+@media (--tablet-and-down) {
+  .bleedWrapper {
+    grid-template-columns: 40px 1fr 40px;
+  }
+}
+
+
+/* Main grid */
+
+.gridParent {
+  column-gap: var(--grid-column-gap);
+  display: grid;
+  grid-column-start: 2;
+  grid-template-columns: repeat(var(--grid-column-count), minmax(0, 1fr));
+}
+
+.gridParent > * {
+  min-width: 0;
+}
+
+@media (--tablet-and-down) {
+  .gridParent {
+    display: block;
+    padding-top: 30px;
+  }
+}
+
+
+/* Main grid areas */
+
+.gridAreaLeft {
+  grid-column: 2 / span 6;
+  grid-row: 1;
+}
+
+.gridAreaRight {
+  grid-column: 9 / span 4;
+  grid-row: 1;
+}
+
+
+/* Left grid area */
+
+.titleWrapper {
+  font: var(--font-headline);
+  margin-top: 90px;
+}
+
+.subtitleWrapper {
+  font: var(--font-body-large);
+  margin-top: 15px;
+}
+
+.descriptionWrapper {
+  font: var(--font-body-medium);
+  margin-top: 20px;
+}
+
+.ctaButtonWrapper {
+  margin-bottom: 83px;
+  margin-top: 30px;
+}
+
+@media (--tablet-and-down) {
+  .titleWrapper {
+    margin-top: 30px;
+  }
+
+  .ctaButtonWrapper {
+    margin-bottom: 0;
+  }
+}
+
+
+/* Right grid area*/
+
+.image {
+  height: auto;
+  max-width: 100%;
+}
+
+.image1Wrapper {
+  margin-top: 50px;
+  position: relative;
+}
+
+.image2Wrapper {
+  float: right;
+  left: 75px;
+  margin-bottom: 80px;
+  position: relative;
+  top: 271px;
+}
+
+@media (--tablet-and-down) {
+  .image1Wrapper {
+    margin-top: 0;
+    right: 40px;
+  }
+
+  .image2Wrapper {
+    left: 40px;
+    margin: 38px 0 40px 85px;
+    top: 0%;
```

Comment:
You probably don't need this combination of properties, since many of them are trying to push and pull the element along the same dimensions. While there are occasionally times when you may want to simultaneously set margins and relative positions for edges of the bounding box like this, it's usually best to try to stick with one method so that it's simpler to understand.

I also see that you're trying to unset some of the properties from the non-mobile view. Since the two views are trying to do very different things, it might be a good idea to wrap the desktop styles in its own media query (`@media (--laptop-and-up)`) so that we don't have to unset these here.